02-JS-Intro-jQuery/Activity 11 to 13/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title> JS </title>
+	<meta charset="UTF-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link HREF="css/style.css" rel="stylesheet" />
+	<script src="node_modules/jquery/dist/jquery.min.js"> </script>
```

Comment:
Good thing to install jQuery through npm. However please create a package.json file and save the jquery version you want to be installed.